apps/web/src/components/TextArea.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+interface TextAreaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  error?: boolean;
+}
+
+function TextArea(props: TextAreaProps) {
+  const [inputLen, setInputLen] = useState(String(props.value ?? '')?.length || 0);
+
+  const onChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputLen(e.target.value.length);
+    props.onChange && props.onChange(e);
```

Comment:
```suggestion
    props?.onChange?.(e);
```

옵셔널 체이닝으로 축약할 수 있을 거 같아요 ~